localPackages/site_developer/composer.json

```diff
@@ -0,0 +1,40 @@
+{
+    "name": "ttn/site-developer",
+    "description": "Sitepackage for developer training",
+    "type": "typo3-cms-extension",
+    "keywords": [
+        "typo3",
+        "example",
+        "extension"
+    ],
+    "license": "GPL-2.0-or-later",
+    "authors": [
+        {
+            "name": "TYPO3 Trainer Network",
+            "email": "trainer-network@typo3.org"
+        }
+    ],
+    "support": {
+        "issues": "https://github.com/typo3-trainer-network/typo3-distribution/issues",
+        "source": "https://github.com/typo3-trainer-network/typo3-distribution"
+    },
+    "require": {
+        "php": "~7.2 || ~7.3 || ~7.4",
+        "typo3/cms-core": "^9.5"
```

Comment:
Also Fluid, frontend and FSC AFAICS.

Reply:
Until now I've not added those dependencies to the sitepackages again, as they are already part of the main package, and sitepackaes only live in this context.

Should I still add them, or even remove `typo3/cms-core`?

Reply:
The packages that are for the BE and the installation itself (core etc.) should be required in the main `composer.json`, and the packages required for the FE rendering should be required in the site package's `composer.json` (even if there are duplicate entries).